server/build_without_gradle.sh

```diff
@@ -1,4 +1,4 @@
-#!/bin/bash
+#!/usr/bin/env bash
```

Comment:
Ref #426

OK, I don't know which one is good practice (`/bin/bash` seems less risky to me, but why not).

Reply:
It depends of the portability you want to have here. I do think it's a good practice for 2 major reasons:

1. All Linux distributions I'm aware of are compatible with the approach I described in the patch.

2. Some BSD-based distributions like FreeBSD doesn't support your approach. See here: https://forums.freebsd.org/threads/bash-not-found.31849/ .

The `env` binary simply search for the executable in the provided PATH environment variable, so, here, you rely on it instead of providing the full path.

I didn't noticed the previous patch you referred, so if you're good with this, I can rebase and drop it.